contrib/download-frozen-image.sh

```diff
@@ -42,6 +42,8 @@ while [ $# -gt 0 ]; do
 	[ "$tag" != "$imageTag" ] || tag='latest'
 	tag="${tag%@*}"
 
+	imageFile="${image////_}" # "/" can't be in filenames :)
```

Comment:
If it really becomes troublesome we can advise to install a more recent version via Homebrew or something, of course (LOL, or follow Jess's tutorial on installing Linux)